scripts/delivery.json

```diff
@@ -63,8 +80,21 @@
             "401": { "type": "unit" },
             "404": { "type": "unit" }
           }
+        },
+        
+        {
+          "method": "POST",
+          "body": {
+	    "type": "delivery_form"
+	  },
+          "responses": {
+            "200": { "type": "delivery" },
+            "401": { "type": "unit" },
+            "404": { "type": "unit" },
+            "422":{ "type": "[error]" }
```

Comment:
why is this a sequence of errors?

Reply:
maine apibuilder.io se reference liya tha, so I thought ki aise hi dena hoga.
Will change it to single error.

Reply:
Why is this not updated yet?

Reply:
local pr kr diya tha, push krna bhul gya tha. sorry for that.